Code/GraphMol/Depictor/Wrap/testDepictor.py

```diff
@@ -54,6 +54,8 @@ def compareCoords(m, molFile):
         pos = ci.GetAtomPosition(i)
         opos = co.GetAtomPosition(i)
         if not ptEq(pos, opos):
+            print(Chem.MolToMolBlock(m))
```

Comment:
Perhaps also a string indicating what the error is here?

Reply:
I thought about that, but it's likely that there are going to be a lot of mismatched atom positions when this happens and this would only show the first. I'm 99% sure that you're going to have to look at the structure anyway in order to figure out what's going on, so it just didn't seem worth it.
I can add it if you really think it'll be helpful

Reply:
I agree it probably won't be too useful, but I like a bit of verbiage when reporting test errors.